modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -327,6 +327,53 @@ def ip(self):
             self._ip_cache = ray.get(self._ip_cache)
         return self._ip_cache
 
+    def split(self, split_func, num_splits, *args):
+        """
+        Split the object wrapped by the partition into multiple partitions.
+
+        Parameters
+        ----------
+        split_func : Callable[pandas.DataFrame, List[Any]] -> List[pandas.DataFrame]
+            The function that will split this partition into multiple partitions.
+        num_splits : int
+            The number of resulting partitions (may be empty).
+        *args : List[Any]
+            Arguments to pass to ``split_func``
+
+        Returns
+        -------
+        list
+            A list of partitions.
+        """
+        logger = get_logger()
+        logger.debug(f"ENTER::Partition.split::{self._identity}")
+
+        @ray.remote(num_returns=num_splits)
+        def _split_df(df, split_func, *args):  # pragma: no cover
+            return split_func(df, *args)
+
+        outputs = _split_df.remote(self._data, split_func, *args)
+        logger.debug(f"SUBMIT::_split_df::{self._identity}")
+        logger.debug(f"EXIT::Partition.split::{self._identity}")
+        return outputs
+
+    @classmethod
+    def put_splits(cls, splits):
+        """
+        Create a new partition that wraps the input splits after concatenating them.
+
+        Returns
+        -------
+        PandasOnRayDataframePartition
+            New `PandasOnRayDataframePartition` object.
+        """
+        logger = get_logger()
+        logger.debug(f"ENTER::Partition.put_splits")
```

Comment:
Why do we not have the same logging methods for the dask method?

Reply:
I think it's okay to leave here, as we already have logging at the Ray partition level. https://github.com/modin-project/modin/pull/4587 is more about logging execution (`@ray.remote` decorated functions and Dask).

Reply:
logging hasn't been extended to Dask yet, so I didn't include it. Happy to include it if folks think it could be a good start to building out logging on Dask!